docs/schema/pub_chain_minimal.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Would it be pedantic to define $schema in all schemas?

Reply:
Done,